### DISCUSSION TITLE
I'm encountering a 403 Forbidden error when attempting to edit the content on a DNN page

### DISCUSSION THREAD

**Discussion by suvarnaashoka** [dnnsoftware/Dnn.Platform#5888] [General]
"I am currently facing this issue with DNN, and I'm seeking solutions or guidance from anyone who might have experience in resolving a 403 error while trying to edit content on a DNN page. Any insights or assistance would be greatly appreciated.

**Comment by Timo-Breumelhof** [comment]
What module, the HTML module?
As what kind of user are you logged in? (what role)

**Comment by suvarnaashoka** [comment]
yes, HTML module 

{mysite}/{page}/ctl/Edit/mid/1136?popUp=true 403 forbidden.

And Logged in as superuser.

**Comment by Timo-Breumelhof** [comment]
Ok, did you check if the NTFS rights are set correctly for the file structure on the server?
BTW, a better place to ask these kind of questions is often https://dnncommunity.org/forums

**Comment by suvarnaashoka** [comment]
At times, it succeeds, but there are instances when it encounters failures. The problem is not consistently occurring.

**Comment by valadas** [comment]
What version of DNN and have you tried to see if it still happens when popups are disabled

**Comment by suvarnaashoka** [comment]
DNN Version 9.2.2

**Comment by valadas** [comment]
Well, I would start by upgrading to the latest version to see if that fixes it, a lot of small fixes were done in the last 4 years, so maybe this is fixed already for your use case...

**Comment by suvarnaashoka** [comment]
In my scenario, I upgraded the jQuery version from 3.2.1 to 3.5.0 quite some time ago. Recently, we also upgraded the EasyDNNNews module. Could this be the cause of the issue I'm experiencing with inconsistent editing?

**Comment by valadas** [comment]
Really don't have enough information to know that. I would start by upgrading DNN and 3rd party modules. If the issue is gone, great, if not, well, at least you got other bugfixes, better performance and security, etc. Even if you pinpoint the issue in 9.2.2, there won't be a new 9.2.2, sooo might as well start with that.
